docker/metadata/docker-compose.yml

```diff
@@ -111,6 +111,11 @@ services:
       # Airflow SSL Configurations
       AIRFLOW_VERIFY_SSL: ${AIRFLOW_VERIFY_SSL:-"no-ssl"}
       AIRFLOW_SSL_CERT_PATH: ${AIRFLOW_SSL_CERT_PATH:-""}
+      # ElasticSearch Configurations
```

Comment:
Can you please also group `ELASTICSEARCH_HOST` env that is placed at Line 51 here ?